src/index.js

```diff
@@ -8,9 +8,9 @@ import Chart from './core/core.controller';
 import helpers from './helpers/index';
 import _adapters from './core/core.adapters';
 import Animation from './core/core.animation';
-import Animator from './core/core.animator';
+import {default as Animator} from './core/core.animator';
```

Comment:
I'm surprised you need to change this line

Reply:
That is unrelated, eslint was complaining about it, because there is a `Animator` named export there too.